src/applications/static-pages/translation.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable no-console */
+import recordEvent from 'platform/monitoring/record-event';
+
+const faqI18Content = {
+  en: {
+    linkTitle: 'Read this page in English',
+    langToggleLink: 'coronavirus-veteran-frequently-asked-questions',
+  },
+  es: {
+    linkTitle: 'Leer esta página en Español',
+    langToggleLink: 'coronavirus-veteran-frequently-asked-questions-esp',
+  },
+};
+
+const configureTranslationLink = (e, currentLang, targetLang) => {
+  const contentDiv = document.getElementById('content');
+  contentDiv.lang = currentLang;
+  e.dataset.lang = currentLang;
+  e.innerText = faqI18Content[targetLang].linkTitle;
+  e.href = faqI18Content[targetLang].langToggleLink;
+  e.onclick = recordEvent({
+    event: 'faq-lang-toggle',
+    targetLang,
+  });
```

Comment:
I think you want to track the event when the link is clicked, not as soon as it is shown -
```suggestion
  e.onclick = () => recordEvent({
    event: 'faq-lang-toggle',
    targetLang,
  });
```